README.md

```diff
@@ -1,4 +1,10 @@
-# UCAN Invocation Specification v1.0.0-rc.1
+# UCAN Invocation Specification v1.0.0-rc. 1
+
+FIXME
+- move pipelines, await, etc to own spec
+- explain in FAQ why we don't need `join` (i.e. use promises, since the model is inverted)
+- add expiry
+- Consider ability heirarcy as being subpaths: `fs/mutate/write`
```

Comment:
Issuing a UCAN with e.g., `fs` would be kinda akin to issuing UCAN that bundles multiple single capabilities (`fs` `mutate` write`).
From what I heard, such UCANs are harder to manage due to lack of fine-granularity.
[It's been reported](https://github.com/ucan-wg/spec/issues/171#issuecomment-1733959444) that having single capability per UCAN is a handy approach. Just food for thought

Reply:
It's true, but also assuems that all agents are live processes. Adding new messages to a heirarchy ina bardwards-compatible way after the fact in an API is standard practice. Without this mechanism, you would need to issue new UCANs to every existing delegation chain.

Fission and DAG House both have use cases where the original Agent burned their keys after issuing, or is kept in cold storage. Reissuing is impossible in these cases.